tools/tool-env/isopackets.js

```diff
@@ -230,7 +230,7 @@ var newIsopacketBuildingCatalog = function () {
         localPackageSearchDirs: [
           files.pathJoin(files.getCurrentToolsDir(), 'packages'),
           files.pathJoin(files.getCurrentToolsDir(),
-            'packages-for-isopackets', 'blaze', 'packages')
+            'packages', 'non-core', 'blaze', 'packages')
```

Comment:
@tmeasday I would prefer not to hard-code specific package/project names like `blaze` in tool code, though I'm not sure the best way to do that. Thoughts?